restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/telephony/UssdCallManager.java

```diff
@@ -282,7 +282,7 @@ private ActorRef outbound(final Object message) throws ServletParseException {
         SipURI to = (SipURI)sipFactory.createSipURI(request.to(), uri);
 
         String transport = (to.getTransportParam() != null) ? to.getTransportParam() : "udp";
-        from = outboundInterface(transport);
+        //from = outboundInterface(transport);
```

Comment:
@croufay if purpose here is to prohibit null assignment in `from` field, then we can do something like this:
```
SipURI obi = outboundInterface(transport);
from = (obi == null) ? from : obi;
```
this way we don't disturb existing logic for execution of `outboundInterface` Method
so if result from `outboundInterface` is null we keep existing value of `from` otherwise we assign `obi` value